packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -23,7 +25,7 @@ describe("ActionMenu", () => {
         unmountAll();
     });
 
-    it("closes/opens the menu on mouse click, space, and enter", () => {
+    it.only("closes/opens the menu on mouse click, space, and enter", () => {
```

Comment:
You'll need to remove this `only` otherwise the other tests in this file won't run.

Reply:
sorry, didn’t catch this